.vscode/settings.json

```diff
@@ -37,5 +37,6 @@
             "pattern": "**/*.spec.ts",
             "command": "yarn test:theia"
         }
-    ]
+    ],
+    "editor.detectIndentation": false
```

Comment:
it breaks the formatting of json files, there 2 spaces are used

Reply:
Yeah, that's annoying. OTOH, I had problems with imported TS files which were detected to have TABS such that on save, space indentations were always converted to TABS causing the linter to fail.

I'll remove this again.